Broken with Helm 3.7.0 Release

Hi, 
first of all, thank you for providing this Action.

**Describe the bug**
With [Helm Release 3.7.0](https://github.com/helm/helm/releases/tag/v3.7.0) the `helm chart` command was removed or partially replaced by other commands:

># Notable Changes
>A few major changes were made to the experimental OCI feature:
>- `helm chart export` has been removed
>- `helm chart list` has been removed
>- `helm chart pull` has been replaced with `helm pull`
>- `helm chart push` has been replaced with `helm push`
>- `helm chart remove` has been removed
>- `helm chart save` has been replaced with `helm package`

Since GitHub is already shipping Helm 3.7.0 with their latest [Ubuntu 20.04 virtual-environment](https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20210919.1), running the `helm-oci-chart-releaser` fails with the following error:

```bash
Error: unknown command "chart" for "helm"
Run 'helm --help' for usage.
Error: Process completed with exit code 1.
```

**To Reproduce**
Steps to reproduce the behavior:
1. Define GitHub workflow using  GitHub virtual-environment `ubuntu-20.04`
2. Define `appany/helm-oci-chart-releaser@v0.2.0` as a wotkflow step

Or checkout one of our repository's workflow (just default stuff):
- [Action Log](https://github.com/carbynestack/ephemeral/runs/3731725362?check_suite_focus=true)
- [Workflow definition](https://github.com/carbynestack/ephemeral/blob/master/.github/workflows/publish-and-release.yaml#L139)

**Expected behavior**
The action should complete (upload the chart) as intended.

**Screenshots**
![image](https://user-images.githubusercontent.com/43000614/135217243-bce87a3e-c9d7-4570-a7b9-dbbf37f8b89a.png)

--- a/README.md
+++ b/README.md
@@ -22,8 +22,7 @@
     registry_username: ${{ secrets.REGISTRY_USERNAME }}
     registry_password: ${{ secrets.REGISTRY_PASSWORD }}
 
-# helm chart pull ghcr.io/appany/my-chart:0.1.0
-# helm chart export ghcr.io/appany/my-chart:0.1.0
+# helm pull ghcr.io/appany/my-chart:0.1.0
 ```
 
 - Push Helm Chart to **Azure Container Registry**
@@ -39,8 +38,7 @@
     registry_username: ${{ secrets.REGISTRY_USERNAME }}
     registry_password: ${{ secrets.REGISTRY_PASSWORD }}
 
-# helm chart pull appany.azurecr.io/helm/my-chart:0.1.0
-# helm chart export appany.azurecr.io/helm/my-chart:0.1.0
+# helm pull appany.azurecr.io/helm/my-chart:0.1.0
 ```
 
 ## Inputs
@@ -83,16 +81,18 @@ outputs:
 
 1. Add container registry and enable OCI support
 ```yaml
-repositories: |
-  - url: ghcr.io
+repositories:
+  ghcr:
+    url: ghcr.io
     type: helm
-    name: appany
-    enableOCI: true
+    name: ...
+    enableOCI: "true"
 
-repository.credentials: |
-  - url: ghcr.io
-    passwordSecret: ...
-    usernameSecret: ...
+credentialTemplates:
+  ghcr:
+    url: ghcr.io
+    username: ...
+    password: ...
 ```
 
 2. Create an Application

--- a/action.yaml
+++ b/action.yaml
@@ -36,26 +36,18 @@ runs:
     - name: Helm | Login
       shell: bash
       run: echo ${{ inputs.registry_password }} | helm registry login -u ${{ inputs.registry_username }} --password-stdin ${{ inputs.registry }}
-      env:
-        HELM_EXPERIMENTAL_OCI: '1'
 
     - name: Helm | Package
       shell: bash
-      run: helm package ${{ inputs.path == null && format('{0}/{1}', 'charts', inputs.name) || inputs.path }} ${{ inputs.registry }}/${{ inputs.repository }}/${{ inputs.name }}:${{ inputs.tag }}
-      env:
-        HELM_EXPERIMENTAL_OCI: '1'
+      run: helm package ${{ inputs.path == null && format('{0}/{1}', 'charts', inputs.name) || inputs.path }} --version ${{ inputs.tag }}
 
     - name: Helm | Push
       shell: bash
-      run: helm push ${{ inputs.registry }}/${{ inputs.repository }}/${{ inputs.name }}:${{ inputs.tag }}
-      env:
-        HELM_EXPERIMENTAL_OCI: '1'
+      run: helm push ${{ inputs.name }}-${{ inputs.tag }}.tgz ${{ inputs.registry }}/${{ inputs.repository }}/${{ inputs.name }}:${{ inputs.tag }}
 
     - name: Helm | Logout
       shell: bash
       run: helm registry logout ${{ inputs.registry }}
-      env:
-        HELM_EXPERIMENTAL_OCI: '1'
 
     - name: Helm | Output
       id: output

--- a/action.yaml
+++ b/action.yaml
@@ -39,15 +39,15 @@ runs:
       env:
         HELM_EXPERIMENTAL_OCI: '1'
 
-    - name: Helm | Chart
+    - name: Helm | Package
       shell: bash
-      run: helm chart save ${{ inputs.path == null && format('{0}/{1}', 'charts', inputs.name) || inputs.path }} ${{ inputs.registry }}/${{ inputs.repository }}/${{ inputs.name }}:${{ inputs.tag }}
+      run: helm package ${{ inputs.path == null && format('{0}/{1}', 'charts', inputs.name) || inputs.path }} ${{ inputs.registry }}/${{ inputs.repository }}/${{ inputs.name }}:${{ inputs.tag }}
       env:
         HELM_EXPERIMENTAL_OCI: '1'
 
-    - name: Helm | Chart | Push
+    - name: Helm | Push
       shell: bash
-      run: helm chart push ${{ inputs.registry }}/${{ inputs.repository }}/${{ inputs.name }}:${{ inputs.tag }}
+      run: helm push ${{ inputs.registry }}/${{ inputs.repository }}/${{ inputs.name }}:${{ inputs.tag }}
       env:
         HELM_EXPERIMENTAL_OCI: '1'
 

